charts/argo/templates/workflow-aggregate-roles.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Should there be a feature flag around these manifests?

Reply:
Sure, what should it look like in the values file? Here are a few ideas, can also replace `create` with `enable` or `install` if you'd prefer. 

```
aggregateRoles:
  view:
    create: true
  edit:
    create: true
  admin:
    create: true
```

```
createAggregateRoles: true
```

```
aggregateRoles:
  create: true
```

Reply:
I'd suggest a single big flag. KISS

Reply:
Added it as such, please review and resolve conversation if you see fit.